<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
background-color:qlineargradient(spread:pad, x1:0.091, y1:0.101636, x2:0.991379, y2:0.977, stop:0 rgba(209, 107, 165, 255), stop:1 rgba(255, 255, 255, 255));
</string>
  </property>
  <widget class="QListWidget" name="listFilms">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>100</y>
     <width>281</width>
     <height>481</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgba(0,0,0,0);
font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>40</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgba(0,0,0,0);
font: 12pt &quot;MS Shell Dlg 2&quot;;
font: 36pt &quot;MS Shell Dlg 2&quot;; color:rgb(255, 255, 255)</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt;Films&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="back">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:20px;
background-color: rgb(170, 255, 255);
font: 14pt &quot;MS Shell Dlg 2&quot;;

</string>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinRating">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>110</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:20px;
background-color: rgb(170, 255, 255);
font: 14pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="maximum">
    <number>10</number>
   </property>
  </widget>
  <widget class="QPushButton" name="addRating">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>110</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:20px;
background-color: rgb(170, 255, 255);
font: 14pt &quot;MS Shell Dlg 2&quot;;

</string>
   </property>
   <property name="text">
    <string>Add rating</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addRating_2">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>180</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:20px;
background-color: rgb(170, 255, 255);
font: 14pt &quot;MS Shell Dlg 2&quot;;

</string>
   </property>
   <property name="text">
    <string>Remove rating</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
